.github/workflows/rector.yaml

```diff
@@ -53,6 +53,7 @@ jobs:
             -
                  if: github.event.pull_request.head.repo.full_name != github.repository
                  uses: actions/checkout@v2
+                 ref: "fix-rector-config-prefer-this"
```

Comment:
Temporary use ref this PR branch to allow the updated rector.php config run on the packages rector run workflow. It needs to be reverted after merge.